lib/validators.js

```diff
@@ -27,7 +27,7 @@ exports.token = Joi.string()
 
 exports.scope = Joi.string()
   .max(256)
-  .regex(/^[a-zA-Z0-9 _:]+$/);
+  .regex(/^[a-zA-Z0-9 _\/.:]+$/);
```

Comment:
Seeing this change made we realize that this could interact badly with or logic for checking "profile:write" scopes here:

  https://github.com/mozilla/fxa-oauth-server/blob/master/lib/scope.js#L35

Reply:
In particular the "this was a weird scope. don't try to fix it, just say NO!" part of that logic.

Reply:
@rfk are you worried about something like this `https://identity:write.mozilla.org/apps/notes:write` or something else?

Reply:
I didn't have a specific concern in mind, just that the ":" in "https://" might cause weird edge-cases here that we have to watch out for.